patches/src/main/kotlin/app/revanced/patches/pandora/skips/UnlimitedSkipsHookPatch.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.pandora.skips
```

Comment:
For consistency put this in misc like its done for other apps too.
```suggestion
package app.revanced.patches.pandora.misc
```